src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -161,6 +161,7 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
 
     val submission1 = createTestSubmission(workspace, methodConfig, indiv1, Seq(sample1, sample2, sample3))
     val submission2 = createTestSubmission(workspace, methodConfig2, indiv1, Seq(sample1, sample2, sample3))
+    val submission3 = createTestSubmission(workspace, methodConfig2, pair1, Seq(pair1))
```

Comment:
Submission that references the pair in the `GraphEntityDAOSpec` above. Makes sure an edge exists so it can vanish if the edge preserving code doesn't work.